src_saleforce_packages/scratchorgpool/sfdx-project.json

```diff
@@ -1,19 +1,18 @@
 {
-    "packageDirectories": [
-        {
-            "path": "force-app",
-            "default": true,
-            "package": "sfpower-scratchorg-pool",
-            "versionName": "sfpower-pool 1.1.0",
-            "versionNumber": "1.1.0.NEXT"
-        }
-    ],
-    "namespace": "",
-    "sfdcLoginUrl": "https://login.salesforce.com",
-    "sourceApiVersion": "51.0",
-    "packageAliases": {
-        "sfpower-scratchorg-pool": "0Ho1P000000blK7SAI",
-        "sfpower-scratchorg-pool@1.0.0-1": "04t1P000000gOkXQAU",
-        "sfpower-scratchorg-pool@1.1.0-1": "04t1P000000gOqzQAE"
+  "packageDirectories": [
+    {
+      "path": "force-app",
+      "default": true,
```

Comment:
What happened here? there should not be any change in this file

Reply:
yeah there shouldn't  be i must of missed this when i was resolving a merge error from my end on this branch i will revert this.